hummingbot/connector/exchange/zebpay/zebpay_api_order_book_data_source.py

```diff
@@ -0,0 +1,369 @@
+import asyncio
+import aiohttp
+import logging
+
+import cachetools.func
+import pandas as pd
+from typing import (
+    Any,
+    AsyncIterable,
+    Dict,
+    List,
+    Optional,
+)
+from decimal import Decimal
+import time
+
+import requests
+import ujson
+import websockets
+from websockets.exceptions import ConnectionClosed
+
+from hummingbot.logger import HummingbotLogger
+from hummingbot.core.data_type.order_book_tracker_entry import OrderBookTrackerEntry
+from hummingbot.core.data_type.order_book_message import OrderBookMessage
+from hummingbot.core.data_type.order_book import OrderBook
+from hummingbot.core.data_type.order_book_tracker_data_source import OrderBookTrackerDataSource
+
+# import with change to get_last_traded_prices
+from hummingbot.core.utils.async_utils import safe_gather
+
+from hummingbot.connector.exchange.zebpay.zebpay_active_order_tracker import zebpayActiveOrderTracker
+from hummingbot.connector.exchange.zebpay.zebpay_order_book_tracker_entry import zebpayOrderBookTrackerEntry
+from hummingbot.connector.exchange.zebpay.zebpay_order_book import zebpayOrderBook
+from hummingbot.connector.exchange.zebpay.zebpay_resolve import get_zebpay_rest_url, get_zebpay_ws_feed, get_throttler
+from hummingbot.connector.exchange.zebpay.zebpay_utils import DEBUG
+
+MAX_RETRIES = 20
+NaN = float("nan")
+
+
+class zebpayAPIOrderBookDataSource(OrderBookTrackerDataSource):
```

Comment:
Classes always start with upper case (because they are definitions of types), objects start with lowercase, because they are instantiation of a certain type... 
Anyway, let's change it to: `ZebpayAPIOrderBookDataSource`